label_maker/validate.py

```diff
@@ -28,5 +28,6 @@
     'imagery': {'type': 'string', 'required': True},
     'background_ratio': {'type': 'float'},
     'ml_type': {'allowed': ['classification', 'object-detection', 'segmentation'], 'required': True},
-    'seed': {'type': 'integer'}
+    'seed': {'type': 'integer'},
+    'imagery_offset': {'type': 'list', 'schema': {'type': 'integer'}}
```

Comment:
Is there a way to validate the list length within cerberus? Also, can you accept any length-2 iterable (e.g., tuple)?

Reply:
I'll add a length test. Tuples aren't really valid JSON (not that we actually check the config for that) so I'd prefer list only